reana_server/config.py

```diff
@@ -145,25 +145,8 @@ def _is_valid_rate_limit(rate_limit: str) -> bool:
 
 
 def _get_rate_limit(env_variable: str, default: str) -> str:
-    env_value = os.getenv(env_variable)
-
-    if not env_value:
-        logging.warning(
-            f"API rate limit config {env_variable} environment variable is not present. "
-            f"Using default value: {default}"
-        )
-        # assuming default has correct format
-        return default
-
-    if _is_valid_rate_limit(env_value):
-        return env_value
-    else:
-        logging.warning(
-            f"API rate limit config {env_variable} environment variable has incorrect format. "
-            f"Incorrect value: {env_value}. Using default value: {default}"
-        )
```

Comment:
The problem with this warning is that it keeps popping after every `reana-admin` command use. Motivation of removing it can be found here: https://mattermost.web.cern.ch/it-dep/pl/gbcszsb7hfncpqx9kus478os8y

Reply:
Ah OK, I thought that the one that was popping all the time was `"API rate limit config {env_variable} environment variable is not present. "Using default value: {default}"`, not both.